package.json

```
@@ -101,8 +106,10 @@
     "test": "npx jest",
```

We want both to run for unit tests:
```suggestion
    "test": "npm run test-jest && npm run test-karma",
```